### DISCUSSION TITLE
Domain folder not removed after deletion through UI

### DISCUSSION THREAD

**Discussion by binarydad** [modoboa/modoboa#2454] [Help]
Basically, I made a domain, and a couple users as a test, and deleted it via the modoboa UI. However, I still see the folder, and accounts, under `/srv/vmail`. Can I manually delete this folder? Should a CRON job take care of it? It's been about a day. Thanks.

**Comment by MrGeneration** [comment]
Sounds like the issue I had previously:
https://github.com/modoboa/modoboa/issues/2157

I could solve it - TL;DR:
- https://github.com/modoboa/modoboa/issues/2157#issuecomment-814754475
- https://github.com/modoboa/modoboa/issues/2157#issuecomment-905664562

Please have a look if this solves your problem as well.

**Comment by binarydad** [comment]
Can I just delete the folder manually or is that unsafe? Thanks.
